src/frontend/components/UI/ToggleSwitch/index.tsx

```diff
@@ -21,12 +22,13 @@ export default function ToggleSwitch(props: Props) {
     title,
     htmlId,
     extraClass,
-    description = ''
+    description = '',
+    fading
   } = props
   const { isRTL } = useContext(ContextProvider)
 
   return (
-    <>
+    <span className={classNames()}>
```

Comment:
Won't this `classNames()` call just give you an empty string?

Reply:
Hmm, I think something was there and I removed the class and forgot the call. good catch